### PR TITLE
Align to new system properties

### LINKED ISSUE #120: Update - System Property change
see CloudSlang/cloud-slang#538

### PR DESCRIPTION
**Pull request by SamMarkowitz**
fixes #120

Note: will fail build until new builder is ready and builder version is updated here.

### CODE REVIEW

File: cloudslang_cli.rst [hunk 1]
```diff
@@ -221,15 +222,25 @@ paths.
...
-      - three
-    examples.properties.map:
-      one: a
-      two: b
-      three: c
+    namespace: examples.sysprops
+
+    properties:
+      - host: 'localhost'
+      - port: 8080
```
Comment (levice14) [inline cloudslang_cli.rst]: map here, without `-`

File: tutorial/13_lesson.rst [hunk 2]
```diff
@@ -10,53 +10,63 @@ values of inputs.
...
-    tutorials.hiring.hostname: <host name>
-    tutorials.hiring.port: '<port>'
-    tutorials.hiring.system_address: <system email>
-    tutorials.hiring.hr_address: <hr email>
+    namespace: tutorials.properties
+
+    properties:
+      domain: bcompany.com
+      hostname: <host>
+      port: <25>
```
Comment (levice14) [inline tutorial/13_lesson.rst]: do we need the `<>` here?

File: tutorial/13_lesson.rst [hunk 3]
```diff
@@ -66,73 +76,25 @@ properties file.
...
 .. code-block:: yaml
 
     - send_mail:
         do:
           mail.send_mail:
-            - hostname
-            - port
-            - from
-            - to
+            - hostname: ${get_sp(tutorials.properties.hostname)}
```
Comment (levice14) [inline tutorial/13_lesson.rst]: you'll need quotes here for the key

File: tutorial/13_lesson.rst [hunk 4]
```diff
@@ -310,15 +269,19 @@ New Code - Complete
...
-    tutorials.hiring.hostname: <host name>
-    tutorials.hiring.port: '<port>'
-    tutorials.hiring.system_address: <system email>
-    tutorials.hiring.hr_address: <hr email>
+    namespace: tutorials.properties
+
+    properties:
+      domain: bcompany.com
+      hostname: <host>
+      port: <25>
```
Comment (levice14) [inline tutorial/13_lesson.rst]: same here for `<>`

**Comment by levice14** [comment]
:+1:

**Comment by SamMarkowitz** [comment]
Waiting for new builder.

**Comment by levice14** [comment]
```
1:17:38 [INFO] 87 .sl files were found
11:17:38 [INFO] 
11:17:40 [INFO] Ignoring file: bcompany.sl (not a valid executable). Top level key 'properties' found instead of 'flow'/'operation'.
11:17:40 [INFO] Ignoring file: bcompany.sl (not a valid executable). Top level key 'properties' found instead of 'flow'/'operation'.
11:17:40 [INFO] Ignoring file: bcompany.sl (not a valid executable). Top level key 'properties' found instead of 'flow'/'operation'.
11:17:40 [INFO] Compiled: 'examples.paths.flow' successfully
11:17:40 [INFO] Compiled: 'tutorials_13.hiring.order' successfully
11:17:40 [INFO] Compiled: 'examples.defaultnav.nav_flow' successfully
...
11:17:40 [INFO] 
11:17:40 [INFO] ------------------------------------------------------------
11:17:40 [INFO] BUILD SUCCESS
11:17:40 [INFO] ------------------------------------------------------------
11:17:40 [INFO] Found 84 slang files under directory: "/home/ubuntu/docs" and all are valid.
```

build passes with new builder

**Pull request merged**